src/ecpool_worker.erl

```diff
@@ -49,8 +49,8 @@
           id :: pos_integer(),
           client :: pid() | {pid(), pid()} | undefined,
           mod :: module(),
-          on_reconnect :: ecpool:conn_callback() | [ecpool:conn_callback()] | undefined,
-          on_disconnect :: ecpool:conn_callback() | [ecpool:conn_callback()] | undefined,
+          on_reconnect :: [ecpool:conn_callback()],
```

Comment:
```suggestion
          on_reconnect = [] :: [ecpool:conn_callback()],
```

Reply:
`ensure_callback` in `init` already takes care of that?